posdealers/04-after-sales/troubleshooting-hardware-tse-reklamation

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
This file should is missing the file extension: the file name currently is `troubleshooting-hardware-tse-reklamation`, but it should be `troubleshooting-hardware-tse-reklamation.md`